apps/src/StudioApp.js

```diff
@@ -2855,6 +2865,19 @@ StudioApp.prototype.polishGeneratedCodeString = function (code) {
 };
 
 /**
+ * Returns whether this view should be responsive based on level options.
+ * Responsive means that the visualizationColumn should resize as the
+ * window width changes, and that a grippy is available to manually resize
+ * the visualizationColumn.
+ */
+StudioApp.prototype.isResponsiveFromConfig = function (config) {
+  // use config.responsiveEmbedded because this.responsiveEmbedded may not be initialized yet
```

Comment:
Adding code (`this. responsiveEmbedded`) that isn't used anywhere will just lead to confusion imo.

AFAICT, `this.centerEmbedded` is only used in one place, where it could probably also instead just used `config.centerEmbedded`? Might be worth cleaning that up too.